website/tutorial/basic/02-getting-started.mdx

```diff
@@ -0,0 +1,325 @@
+- `schema` - a combination of the GraphQL SDL and the resolvers. `makeExecutableSchema` function is in charge of gluing them together into an executable schema we can later use.
+
+Now that you have a GraphQL schema, you can use that to fetch data using a GraphQL `query` operation!
+
+### Query the GraphQL schema
+
+As explained before, the GraphQL schema is only your contract, and it exposes the set of all types and capabilities that your API layer can do.
+
+To use your GraphQL schema and consume data from it, you will need to write an executable GraphQL `query` definition.
+
+Based on the schema your created before, you can use the following query:
+
+```graphql
+query {
+  info
+}
+```
+
+So you don't have to get into all the complexity of running GraphQL server - you can use this query and run it against your GraphQL schema and get an immediate result.
+
+To query our local schema, even without any fancy GraphQL client or even a GraphQL server, you can use GraphQL's `execute` function to just run the schema with the query.
+
+<Instruction>
+
+Update the code in `src/main.ts` to contain the following snippet:
+
+```ts
+import { execute, parse } from 'graphql'
+import { schema } from './schema'
+
+async function main() {
+  const myQuery = parse(/* GraphQL */ `
+    query {
+      info
+    }
+  `)
+
+  const result = await execute({
+    schema,
+    document: myQuery,
+  })
+
+  console.log(result)
+}
+
+main()
+```
+
+</Instruction>
+
+Now, try to run our project again (either with `npm run dev` or `npm run start`), you should see in the output log the following:
+
+```
+{ data: [Object: null prototype] { info: 'Test' } }
+```
+
+**So what happened here?**
+
+First, the GraphQL `query` was parsed it using `parse` function of `graphql` - this will create a `DocumentNode` object that can later be executed by GraphQL.
+
+Then, the `execute` function of `graphql` was called with the following parameters:
+
+1. `schema` - this is the GraphQL schema object you previously created.
+2. `myQuery` - this is the `DocumentNode` object created based on our GraphQL query.
+
+The return value of `execute` is the GraphQL result (or, GraphQL response).
+
+The GraphQL engine takes the query, and based on the fields you selected (called the _Selection-Set_), it runs the resolvers, and returns their return value.
+
+The next chapter will teach your how to use your GraphQL schema to create a GraphQL server!
+
+### A word on the GraphQL schema
+
+At the core of every GraphQL API, there is a GraphQL schema. So, let's quickly talk about it.
+
+> **Note**: In this tutorial, we'll only scratch the surface of this topic. If you want to go a bit more in-depth and
+> learn more about the GraphQL schema as well as its role in a GraphQL API, be sure to check out
+> [this](https://www.prisma.io/blog/graphql-server-basics-the-schema-ac5e2950214e) excellent article.
```

Comment:
We should check how up to date this is.
If it is not, we can ask them to update or we can create our own

Reply:
I think the resource is actually pretty good!

(if you ignore this broken quote)

![image](https://user-images.githubusercontent.com/14338007/155740613-75261a96-eb6b-41ee-a5e6-0cd85826ef72.png)